src/styles/spacing/index.js

```diff
@@ -19,17 +19,33 @@ export const spacings = {
   HUGE: 64
 }
 
+const createXRules = (type, value) => ({
+  [`${type}Left`]: value,
+  [`${type}Right`]: value
+})
+
+const createYRules = (type, value) => ({
+  [`${type}Top`]: value,
+  [`${type}Bottom`]: value
+})
```

Comment:
Should we memoize these? It's possible they'll be called very frequently if someone is using them in a `render()`

Reply:
I don't think we need to memoize these because it looks like they are used internally only to generate the spacing object.  That simple spacing object is then what could be called many times by the user.

Reply:
ah yeah, that makes sense :)